app/helpers/link_to_helper.rb

```diff
@@ -254,6 +254,20 @@ def unhappy_url(info_request = nil)
     end
   end
 
+  # Google Analytics
+
+  # if used, value must an integer
+  def analytics_event_code(category, action, label=nil, value=nil, label_is_script=false)
```

Comment:
I have a mild preference for calling this method `track_analytics_event` as its a bit more "Tell, don't ask" (okay, not exactly, but I think better describes what will _happen_ when used, rather than what it _is_). Shrug! :lollipop: